Automattic/woocommerce-payments
# Run E2E tests with Travis

Fixes #721 

#### Changes proposed in this Pull Request

* Setup environment for e2e tests with a script.
* Run e2e tests with `@wordpress-scripts`.
* Update travis config to run e2e tests on each PR.

#### Testing instructions

**Locally** - follow instructions in `tests/e2e/README.md`

**In Travis** - check output of "E2E tests" build step. It should pass successfully.

Successful run: https://github.com/Automattic/woocommerce-payments/pull/746/commits/27f21bb740e5932acc5dbc7ebcd6e581319ea3a1 | [travis details](https://github.com/Automattic/woocommerce-payments/runs/850481415)
Failed run: https://github.com/Automattic/woocommerce-payments/pull/746/commits/8d3e7c83e2613af17a7beb4d392949b3a8166ebd | [travis details](https://github.com/Automattic/woocommerce-payments/runs/850655268)
Slack report: p1594226655417300-slack-CQ0Q6N62D

**TODO:**

- [x] Fix failing test in travis. 
- [x] Restore unit tests in travis and run only on master. 
-------------------
